dev-env/windows/manifests/bazel.json

```diff
@@ -1,12 +1,11 @@
 {
   "homepage": "https://bazel.build",
-  "version": "2.0.0",
+  "version": "2.0.0-patched",
   "license": "Apache-2.0",
   "bin": "bazel.exe",
   "architecture": {
     "64bit": {
-      "url": "https://github.com/bazelbuild/bazel/releases/download/2.0.0/bazel-2.0.0-windows-x86_64.zip",
-      "hash": "83d0e987fa84ac5085c1fe1cc8fd36f5c2bd7b0a3e84b346c2093b19cfa35a57"
+      "url": "D:\patched-bazel\bazel.zip"
```

Comment:
This means that our installation instructions for local development don’t work anymore. Having to manually build Bazel sounds rather annoying and at least for now, I don’t expect us to make significant changes so maybe it would be best to leave the old code and just patch this in our CI job?

Building the Bazel binary here seems to make it pretty hard to have any chance of reusing the cache since the build of Bazel is nondeterministic and it seems like the hash of the Bazel binary influences things. I guess you could download the pipeline artifact from a recent build. Not a big issue since getting the cache to work requires replicating the CI setup almost identically anyway (up to user names, working directories, …) so I tend to just build without the cache on Windows.

Reply:
Binary will now be hosted in a publicly-readable bucket, which means the scoop install script will work locally OOTB as well.